javascript/builtins/Array.json

```diff
@@ -2302,6 +2302,9 @@
                 {
                   "version_added": "0.12",
                   "version_removed": "4.0.0"
+                },
+                {
+                  "version_added": "10.9.0"
```

Comment:
This should be the first element in this array.

Reply:
Could you explain why? For both suggestions. Thanks!

Reply:
It must be the first element in the array as the first element is used as the default view:
https://github.com/mdn/kumascript/blob/master/macros/CompatBeta.ejs#L509-L522

As for keeping `{"version_added":"0.12","version_removed":"4.0.0"}`, that’s because this repository is about having all compatibility data about which features were supported on which platforms and in which versions.

needinfo?(@Elchi3)

Reply:
Thanks! Makes sense.

Another question. The version info array has three elements: [{10.9.0}, {0.12, 4.0.0}, {6.5.0 with flag}]. In what order should the second and third element be? Or maybe that doesn't matter...

Reply:
The order reverse-chronological per version_added, see https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#the-support_statement-object

Reply:
Thanks! Done.